src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -13689,6 +14298,8 @@ public partial interface I1<T> where T : I1<T>
     abstract static T operator " + op + @"(T x, int y);
 }
 
+#pragma warning disable CS0660, CS0661 // 'C1' defines operator == or operator != but does not override Object.Equals(object o)/Object.GetHashCode()
```

Comment:
Did I miss a test that verified that these were actually reported, rather than suppressing or skipping?

Reply:
> Did I miss a test that verified that these were actually reported, rather than suppressing or skipping?

There is nothing special about this diagnostics in relation to the feature. Existing tests should be sufficient.